dev/io.openliberty.org.jboss.resteasy.mprestclient/src/io/openliberty/microprofile/rest/client30/internal/LibertyProxyClassLoader.java

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
Probably should add the EPL v2 license as well to it since it is a new class you are adding based off of hte other one and it having a new package.  Same with the 4.0 one of course.

Reply:
It's my understanding that I can copy the Apache v2 class and modify it (including renaming and changing the package), but I have to retain the original copyright

Reply:
Yes